docs/src/features.md

```diff
@@ -1,33 +1,18 @@
 
```

Comment:
This page should probably only have a high-level overview of the main features with screencasts. The existing content doesn't feel ideal here.